lib/ssh_scan/database.rb

```diff
@@ -0,0 +1,26 @@
+# API Databases
+require 'ssh_scan/database/mongo'
+require 'ssh_scan/database/sqlite'
+
+module SSHScan
+  class DatabaseConfig
+    def initialize(opts = {})
+      @@db_type = opts['database']['type'] || 'mongodb'
+    end
+
+    def self.from_config_file
+      @@db_path = './lib/ssh_scan/database/database_config.yml'
```

Comment:
Yeah, I think we should have a common place for config file. Something like, `./config/database/` for database config files and similarly for others.